hedwiki/score/HED_score_0.0.1.mediawiki

```diff
@@ -269,32 +269,12 @@ Reference:
         ** <nowiki># {takesValue}[free text]</nowiki>
 
 
-'''Findings-attributes''' <nowiki> [Subtree tree for general properties]</nowiki>
+'''Finding-attributes''' <nowiki> [Subtree tree for general properties]</nowiki>
 
-    * Not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
-    * No-definite-abnormality
-    * Yes
-    * No
+    * Significance <nowiki>[Significance of finding. When normal/abnormal could be labeled with base schema 'Normal'/'Abnormal' tags]</nowiki>
```

Comment:
`Significance` as a standalone might be a problem.  Could we make it `Finding-significance`?

Reply:
Comment on: [hedwiki/score/HED_score_0.0.1.mediawiki](https://github.com/hed-standard/hed-schema-library/pull/15/files/bbf9c6fe37fa3978c1f6d63538ba2a6bc23e6603#diff-cf24cd0715246f155b661872ce60e598ee969745e1e2187c99f1c7a4efb91eb7) and this is addressed in: [HED_score_0.0.2.mediawiki](https://github.com/hed-standard/hed-schema-library/pull/15/files/e0ecc883d1765d9f8e19f3224144afaf21882dbc..8294ec01e20b9792f6344a173efde3b0f052ced7#diff-fa4b84b056608e69db2ff24ea3bab2c93bb3ae956f0f79cab10c4bb9714ea8e7)